chunky/src/java/se/llbit/png/PngFileWriter.java

```diff
@@ -41,9 +40,9 @@
   /**
    * @throws IOException
    */
-  public PngFileWriter(File file) throws IOException {
```

Comment:
This broke the denoiser plugin (and any other plugin that uses the PngFileWriter to save images).